src/collection.ts

```diff
@@ -340,12 +340,17 @@ export class Collection<TSchema extends Document = Document> {
    * Update a single document in a collection
    *
    * @param filter - The filter used to select the document to update
-   * @param update - The update operations to be applied to the document
+   * @param update - The modifications to apply
    * @param options - Optional settings for the command
    */
   async updateOne(
     filter: Filter<TSchema>,
-    update: UpdateFilter<TSchema> | Partial<TSchema>,
+    /**
+     * The value of update can be either:
+     * UpdateFilter<TSchema> - A document that contains update operator expressions,
+     * Document[] - an aggregation pipeline.
+     * */
+    update: UpdateFilter<TSchema> | Document[],
```

Comment:
A TSDoc comment on a parameter like this doesn't render in the docs, you can run `npm run build:docs` and `npm run preview:docs` to see for yourself

Reply:
TIL ✨